### PR TITLE
[aws] Add configuration option for TLD for httpjson inputs

### PR DESCRIPTION
**Pull request by kgeller**
## Proposed commit message

[aws] Update `guardduty`, `inspector`, `securityhub_findings` and `securityhub_insights` httpjson configs to allow the configuration of the TLD. Default will be set to currently hardcoded value of `amazonaws.com`.
## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- 

## Screenshots
New config option under the Advanced Settings 

Screenshot 2023-09-29 at 9 55 39 AM

### CODE REVIEW

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.8.0"
+  changes:
+    - description: Allow configuration of TLD for guardduty, inspector, and security hub datastreams.
+      type: bugfix
```
Comment (bhapas) [inline packages/aws/changelog.yml:5]: `enhancement` ?

Comment (kgeller) [reply packages/aws/changelog.yml:5]: I was torn here, I can update

**Review by bhapas — COMMENTED** [review]
Missed to add this before. Sorry.

Can we add `tld` to the [system test config ](https://github.com/elastic/integrations/blob/main/packages/aws/data_stream/guardduty/_dev/test/system/test-default-config.yml#L8)[ something other than amazonaws.com as well ] and test this?

**Comment by kgeller** [comment]
> Can we add `tld` to the [system test config ](https://github.com/elastic/integrations/blob/main/packages/aws/data_stream/guardduty/_dev/test/system/test-default-config.yml#L8)[ something other than amazonaws.com as well ] and test this?

Added for the securityhub_findings datastream since that was where the issue was reported

File: packages/aws/data_stream/securityhub_findings/_dev/test/system/test-default-config.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ vars:
 data_stream:
   vars:
     aws_region: xxxx
+    tld: amazonaws.cn
```
Comment (bhapas) [inline packages/aws/data_stream/securityhub_findings/_dev/test/system/test-default-config.yml:12]: The deploy has `hostname`

https://github.com/elastic/integrations/blob/50d92120b6cc864ca871adca5587ace8dbf5d617/packages/aws/data_stream/securityhub_findings/_dev/deploy/docker/docker-compose.yml#L5

  `aws_region` being `xxxx`
So, shouldn't `tld` be `amazonaws.com`

Comment (kgeller) [reply packages/aws/data_stream/securityhub_findings/_dev/test/system/test-default-config.yml:12]: Updated!

**Review by bhapas — APPROVED** [review]
LGTM

**Review by zmoog — APPROVED** [review]
LGTM

**Pull request merged**